server/konnectors/currencies.coffee

```diff
@@ -0,0 +1,152 @@
+americano = require 'americano-cozy'
+request = require 'request'
+moment = require 'moment'
+crypto = require 'crypto'
+async = require 'async'
```

Comment:
If I have to run this in my root directory (I suppose that means ssh into my cozy), will that force everyone to do that in their cozy as well? Isn't there a way to add that package to the official cozy distribution?

I added the `package.json` dependency and fixed the code in the temporary follow-up commit "Fix: Add xml2js as dependency."